sql/src/main/java/io/crate/planner/operators/JoinPlanBuilder.java

```diff
@@ -64,23 +66,27 @@
     private final MultiSourceSelect mss;
     private final WhereClause where;
     private final SubqueryPlanner subqueryPlanner;
-    private final SessionContext sessionContext;
+    private final Functions functions;
+    private final TransactionContext txnCtx;
 
     private JoinPlanBuilder(MultiSourceSelect mss,
                             WhereClause where,
                             SubqueryPlanner subqueryPlanner,
-                            SessionContext sessionContext) {
+                            Functions functions,
+                            TransactionContext txnCtx) {
```

Comment:
the abbreviation is a bit far fetched for my taste :D

Reply:
`txn` is a quite common abbreviation for transaction. https://www.allacronyms.com/TXN/Transaction